release-notes/v1.2.0-rc.1.yaml

```diff
@@ -0,0 +1,131 @@
+date: October 25, 2024
+
+# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
+breaking changes: |
+  Gateway API GRPCRoute and ReferenceGrant v1alpha2 have been removed.
+  Please refer to the [Gateway API v1.2.0 documentation](https://github.com/kubernetes-sigs/gateway-api/releases) for more information.
+
```

Comment:
* Removed default CPU limit of the Envoy Gateway deployment (https://github.com/envoyproxy/gateway/pull/4290)
* Default Envoy Shutdown Settings have changed https://github.com/envoyproxy/gateway/pull/4288 & https://github.com/envoyproxy/gateway/pull/4230
*

Reply:
Are these considered breaking? Or worth noticing?